frontend/packages/dev-console/integration-tests/features/topology/quick_search_from_topology_views _to_add_to_project_ODC-2925

```diff
@@ -0,0 +1,73 @@
+Feature: Provide quick search from topology/list views to add to project 
+    As a user, when in Topology List/Graph view, I should be able to have a quick way to search for items to add to my app/project
+
+Background:
+    Given user is at developer perspective
+    And user is "aut-quick-search" namespace
+
+
+@regression
+Scenario: Quick search button in topology chart view
+    Given user is at topology view
+    When user clicks on chart view button
+    Then user can see quick search button
+
+
+@regression
+Scenario: Quick search button in topology list view
+    Given user is at topology view
+    When user clicks on list view button
+    Then user can see quick search button
+
+
+@regression
+Scenario: Quick search bar in topology chart view
+    Given user is at topology chart view
+    When user clicks Add to project button
+    Then user can see Add to project search bar
+
+
+@regression
+Scenario: Quick search bar in topology list view
+    Given user is at topology chart view
+    When user clicks Add to project button
+    Then user can see Add to project search bar
+
+
+@regression
+Scenario: Add django application in topology chart view
+    Given user is at topology chart view
+    When user clicks Add to project button
+    And user type django in Add to project search bar
+    And user select django+PostgreSQL option
+    And user clicks on instantiate template
+    And user clicks on create with default values in Instantiate Template form
+    And user can see postgres and django workload in topology chart view
+
+
+@regression
+Scenario: Add .Net application in topology list view
+    Given user is at topology list  view
+    When user clicks Add to project button
+    And user type "ab" in Add to project search bar
+    And user select .Net core option
+    And user clicks on Create Application
+    And user clicks on create with default values in Create Application form
+    And user can see .Net workload in topology list view
+
+
+@regression
+Scenario: View all results option for django in topology chart view
+    Given user is at topology chart view
+    When user clicks Add to project button
+    And user type django in Add to project search bar
+    And user click on View all results option
+    Then user will see Catalog with django text filter
+
+
+@regression
+Scenario: No results for the search 
+    Given user is at topology chart view
+    When user clicks Add to project button
+    And user type "abcdef" in Add to project search bar
+    And user will see No result
```

Comment:
Will it display any message? If so, can you please add that message here.

Reply:
The message is  `No message`

Reply:
@sanketpathak it says `No results`

Reply:
Updated